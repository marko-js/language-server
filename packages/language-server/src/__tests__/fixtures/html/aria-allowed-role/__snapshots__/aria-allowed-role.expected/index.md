## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <button role="heading" aria-level="3">
    |  ^^^^^^ Fix any of the following:
  ARIA role heading is not allowed for given element
  2 |   header button
  3 | </button>
```

