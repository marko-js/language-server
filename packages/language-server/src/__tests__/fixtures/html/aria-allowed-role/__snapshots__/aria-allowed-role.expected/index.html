<button data-marko-node-id="0" role="heading" aria-level="3">
  header button
</button>